lib/database.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_record'
+require 'erb'
```

Comment:
OK, I've taken another look -- requiring `erb` here is actually fine, there's some rendering happening in database.yml.
(In other `bin` scripts `erb` gets pulled in by `rmt/config`)